framerail/src/lib/deepwell.server.ts

```diff
@@ -0,0 +1,14 @@
+import axios from 'axios';
+
+const DEEPWELL_HOST = process.env.DEEPWELL_HOST || 'localhost';
+const DEEPWELL_PORT = 2747;
+
+export async function ping(): string {
+  // TODO for now, dump as string
+  const response = await axios.put(
+    `http://${DEEPWELL_HOST}:${DEEPWELL_PORT}/api/trusted/ping`,
+     { timeout: 500 },
+  );
+
+  return response.data;
+}
```

Comment:
Axios isn't really needed depending on the complexity of the API, simpler more verbose `fetch` calls could be used. The old web used an OpenAPI JS generator to handle the grunt work here, which you could use. Not important for this PR really.

Reply:
Yeah I figured. I'm not as experienced with the JS ecosystem, and I have no special attachment here, I'm totally fine changing this to be better. Besides this is hardly a full API fetch framework here that we'd lose.